components/datetime/tests/fixtures/tests/styles.json

```diff
@@ -0,0 +1,107 @@
+[
+    {
+        "input": {
+            "locale": "en",
+            "value": "2020-01-21 08:25:07.000",
```

Comment:
Optional: You might as well make these ISO-8601 and replace the space with a `T`